### PR TITLE
Build/add testnet

### PR DESCRIPTION
**Pull request by lucas-tortora**
# Description of change

Add IOTA 2 testnet. Info on https://iotafoundation.atlassian.net/wiki/spaces/IF/pages/2186680255

## Type of change

- Documentation Enhancement

## Change checklist

- [x] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [x] I have performed a self-review of my own changes
- [x] I have made sure that added/changed links still work
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: theme/src/theme/constant.tsx [hunk 1]
```diff
@@ -7,6 +7,15 @@ export const Networks = {
     permaNodeApi: 'https://chronicle.stardust-mainnet.iotaledger.net',
     explorer: 'https://explorer.iota.org/mainnet',
   },
+  iota_2_testnet: {
+    baseToken: 'Testnet Token (no value)',
+    protocol: 'IOTA 2.0',
+    httpRestApi: 'https://api.nova-testnet.iotaledger.net/',
+    eventApi: 'wss://api.nova-testnet.iotaledger.net:443 (MQTT 3.1, /mqtt)',
+    permaNodeApi: 'https://api.nova-testnet.iotaledger.net/',
```
Comment (Dr-Electron) [inline theme/src/theme/constant.tsx:15]: This is the same as the normal API 🤔

File: docs/build/getting-started/networks-endpoints.mdx [hunk 2]
```diff
@@ -116,3 +116,14 @@ This network is subject to occasional resets (no data retention) which are usual
 #### Additional Info
 
 <NetworkInfo.EvmCustom {...Networks['shimmer_testnet']} />
+
+
+## IOTA 2.0 Testnet
+
+[The IOTA 2.0 Testnet](https://explorer.shimmer.network/shimmer-testnet) acts as a test bed for builders without any real world value.
```
Comment (Dr-Electron) [inline docs/build/getting-started/networks-endpoints.mdx:123]:
```suggestion
[The IOTA 2.0 Testnet](https://explorer.iota.org/iota2-testnet) acts as a test bed for builders without any real world value.
```

**Review by Dr-Electron — CHANGES_REQUESTED** [review]

File: theme/src/theme/constant.tsx (hunk 1 above)
Comment (lucas-tortora) [reply theme/src/theme/constant.tsx:15]:
```suggestion
    permaNodeApi: 'https://chronicle.nova-testnet.iotaledger.net',
```

File: docs/build/getting-started/networks-endpoints.mdx [hunk 3]
```diff
@@ -116,3 +116,14 @@ This network is subject to occasional resets (no data retention) which are usual
...
 <NetworkInfo.EvmCustom {...Networks['shimmer_testnet']} />
+
+
+## IOTA 2.0 Testnet
+
+[The IOTA 2.0 Testnet](https://explorer.iota.org/iota2-testnet) acts as a test bed for builders without any real world value.
+
+:::info
+This network is subject to occasional resets (no data retention) which are usually announced with a one-week grace period.
+:::
```
Comment (Dr-Electron) [inline docs/build/getting-started/networks-endpoints.mdx:127]:
```suggestion
:::info

This network is subject to occasional resets (no data retention) which are usually announced with a one-week grace period.

:::
```

**Review by Dr-Electron — APPROVED** [review]

**Pull request merged**